app/jsonimplicits/implicits.scala

```diff
@@ -55,6 +57,7 @@ object model {
     implicit val recurringCharge: Writes[RecurringCharge] = Json.writes[RecurringCharge]
     Json.writes[Reservation]
   }
+  implicit val rdsWriter: Writes[Rds] = Jsonx.formatCaseClass[Rds]
```

Comment:
Scala permits us to have 22 fields in a case class. Our Rds class contained more than that, so we used the `play-json-extensions` library to solve this problem.